packages/web/components/swap-tool/index.tsx

```
@@ -277,676 +320,306 @@ export const SwapTool: FunctionComponent<SwapToolProps> = observer(
+      !isNil(account?.address) &&
+      !swapState.inAmountInput.hasErrorWithCurrentBalanceQuote &&
+      !swapState.inAmountInput?.balance?.toDec().isZero() &&
+      swapState.inAmountInput.isLoadingCurrentBalanceNetworkFee;
+
+    const isConfirmationDisabled =
+      isSendingTx ||
+      isWalletLoading ||
+      (account?.walletStatus === WalletStatus.Connected &&
+        ((quoteType === "out-given-in" && swapState.inAmountInput.isEmpty) ||
+          (quoteType === "in-given-out" && swapState.outAmountInput.isEmpty) ||
+          !Boolean(swapState.quote) ||
```

Biome

[error] Avoid redundant `Boolean` call

It is not necessary to use `Boolean` call when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant `Boolean` call


(lint/complexity/noExtraBooleanCast)